### PR TITLE
url option needed for wp site networks

### PR DESCRIPTION
**Pull request by allenfear**
Ruby caught this here:
https://pantheon-systems.desk.com/web/agent/case/79529

Closes #
n/a

## Effect
PR includes the following changes:
Adding the Terminus command needed for the case of a wp site network

### CODE REVIEW

File: source/_docs/wordpress-redis.md [hunk 1]
```diff
@@ -31,6 +31,13 @@ This method will store object cache values persistently in Redis while preservin
  ```
  terminus wp <site>.<env> -- plugin install wp-redis
  ```
+ 
+ For site networks, you will need to specify the site URL by adding that to the command:
+ 
+  ```
```
Comment (alexfornuto) [inline source/_docs/wordpress-redis.md]: Is there supposed to be a space before these ticks?

Comment (allenfear) [inline source/_docs/wordpress-redis.md]: Yep. See [Terminus command usage](https://pantheon.io/docs/terminus/examples/#module,-theme,-and-plugin-updates). But you should also install Terminus and try it out to confirm.

**Comment by rachelwhitton** [comment]
+1 Thanks @allenfear!

File: source/_docs/wordpress-redis.md (hunk 1 above)
Comment (alexfornuto) [inline source/_docs/wordpress-redis.md]: I meant the codeblock backticks, but it turns out it was GitHub's diff view messing with me. I'm going to delete this comment to hide my shame, but your email will forever remember it.

**Pull request merged**